magenta-lib/src/main/scala/magenta/Project.scala

```diff
@@ -130,14 +131,18 @@ object HostList {
   implicit def hostListAsListOfHosts(hostList: HostList) = hostList.hosts
 }
 
+case class RuntimeResources(reporter: DeployReporter, lookup: Lookup, parameters: DeployParameters, stack: Stack) {
```

Comment:
The best I've been able to think of is `ResolvedDeployParameters`, or `PerStackDeployParameters`. I'd probably err towards  the first.